appium/README.md

```diff
@@ -24,10 +25,22 @@ To run a particular test:
 
 To run all tests: `npm run-script test.live.all` or `npm run-script test.mock.all`
 
+## Building app for testing
```

Comment:
We now have 2 `Building app for testing` sections, let's merge it in 1 section with 2 methods, like this:
```
## Building app for testing
1. Manually compile build from the current code:
- run `bundle exec fastlane build` in `flowcrypt-ios` folder
- it will produce `appium/FlowCrypt.app` for testing
2. Use the latest simulator build:
- copy `FlowCrypt.app` from `/DerivedData/FlowCrypt-.../Build/Products/Debug-iphonesimulator` (In Xcode open Products folder -> FlowCrypt -> Show in Finder) to appium folder.
```